core/src/test/resources/Xor/basic.source

```diff
@@ -0,0 +1,36 @@
+rewrites = [Xor2Either]
+<<< xor 1
+import scala.concurrent.Future
+import cats.data.{ Xor, XorT }
+trait A {
+type MyDisjunction = Xor[Int, String]
+  val r: MyDisjunction = Xor.Right.apply("")
+  val s: Xor[Int, String] = cats.data.Xor.Left(1 /* comment */)
+  val t: Xor[Int, String] = r.map(_ + "!")
+  val nest: Seq[Xor[Int, cats.data.Xor[String, Int]]]
+  val u: XorT[Future, Int, String] = ???
+}
+>>>
+import cats.data.EitherT
+import cats.implicits._
+import scala.util.Either
+import scala.concurrent.Future
+import cats.data.{ Xor, XorT }
+trait A {
+type MyDisjunction = Either[Int, String]
+  val r: MyDisjunction = Right("")
+  val s: Either[Int, String] = Left(1 /* comment */)
+  val t: Either[Int, String] = r.map(_ + "!")
+  val nest: Seq[Either[Int, Either[String, Int]]]
+  val u: EitherT[Future, Int, String] = ???
+}
```

Comment:
What about for comprehensions? Is it necessary to add `.right` on `Xor.flatmap` or does `cats.implicits` pimp up `Either`?